site/en/getstarted/cluster/install_cluster-milvusoperator.md

```diff
@@ -494,14 +437,25 @@ $ kubectl delete mc my-release
 
 </div>
 
+## Uninstall Milvus Operator
 
-## Delete the K8s cluster
+Run the following command to uninstall Milvus Operator.
```

Comment:
There are also two ways to uninstall Milvus Operator on K8s:

Reply:
[be676e4](https://github.com/milvus-io/milvus-docs/pull/1643/commits/be676e475dedfd1cb36e2ee99d707937bcb26bc6)